src/interactivity/win32/windowio.cpp

```diff
@@ -1003,9 +998,6 @@ DWORD WINAPI ConsoleInputThreadProcWin32(LPVOID /*lpParameter*/)
         // -- END LOAD BEARING CODE
     }
 
-    // Free all resources used by this thread
-    DeactivateTextServices();

```

Comment:
Ideally, yes. However, this code is currently effectively unreachable, because we `TerminateProcess` in the `ServiceLocator` on shutdown. A better location for shutting down TSF is in the destructor of `Window` but I figured it's fine if we just don't. After all, we also got the `delete renderer` calls commented out there.